pre-commit-make-shim

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ex
+
+service=$1
+shift
+make pre-commit-check-in-$service-venv FILES="$*"
```

Comment:
Won't `$*` include `service`? I feel like we don't want that, we want only the files pre-commit is passing.

Reply:
The shift drops the service from `$*`